### PR TITLE
Add link to n_subsample observation on usage page

### PR DESCRIPTION
**Pull request by nilfm99**
Addresses the comment at #1214

### CODE REVIEW

**Comment by kylophone** [comment]
I think maybe the cli tool help is not the right place for this, since there is actually come nuance involved. Would it be better if we put this into some of our usage docs or README?

**Comment by nilfm99** [comment]
got it, I'll update this PR tomorrow

**Comment by nilfm99** [comment]
I thought that documenting the entire `VmafConfiguration` struct was useful and a good place for this information, given that this is a very specific detail, and that our `README.md` already says:

> For complete API documentation, see [libvmaf.h](https://github.com/Netflix/vmaf/blob/master/libvmaf/include/libvmaf/libvmaf.h).

File: libvmaf/include/libvmaf/libvmaf.h [hunk 1]
```diff
@@ -55,6 +55,31 @@ enum VmafPoolingMethod {
...
+ *
+ * @param n_threads   How many threads can be used to run
+ *                    feature extractors concurrently.
+ * 
+ * @param n_subsample Compute scores only every N frames.
+ *                    Note that setting an even value for N can lead to
+ *                    inaccurate results. For more detail, see
+ *                    https://github.com/Netflix/vmaf/issues/1214
+ * 
+ * @param cpumask     Restrict permitted CPU instruction sets.
```
Comment (kylophone) [inline libvmaf/include/libvmaf/libvmaf.h:72]: `cpumask` is actually platform independent, we also have a mask for arm64 neon.

Comment (nilfm99) [reply libvmaf/include/libvmaf/libvmaf.h:72]: sorry, do you mean platform dependent? I can add the NEON comment on the "if cpumask & 1" line

**Comment by kylophone** [comment]
lgtm

**Pull request merged**